docs/v2/ios.md

```diff
@@ -1,53 +1,149 @@
-# iOS Installation
+# React native Agora iOS Installation
+#### Minimum compatibility
+* React native 0.58.x
+* IOS SDK 8.0+
+* Android 5.0+ x86 arm64 armv7a
 
-# Prerequisites
-* You should installed and linked `react-native-agora`
-* Xcode 10.0+
-* cocoapods
+#### 1. open up your terminal and write this :
+When it comes to the version nr you can put any version you like as long as it it compatibility
```

Comment:
I think abbreviation is only comfortable for  native english speaker. probably u need the unfolding the `nr`